tests/compas/files/test_stl.py

```diff
@@ -1,36 +0,0 @@
-import os
```

Comment:
Yes. Moved to test_mesh.py under tests/compas/datastructures.

Reply:
ok! I actually think it is more proper to keep this in a `compas/files/test_stl` because it's supposed to be unit test, and by testing the mesh_from_stl, you're actually already integrating a lot of other things; but in this case, it's ok if you want to keep it in the new location, it's not such a big test anyway.. the only thing I would suggest thou is to split the `test_from_stl` method into different ones, each describing what they are actually testing (now it's only a comment but if it fails, you dont see the comment text in the test failure)

Reply:
Moved to #418.